network/transport/v2/protocol.go

```diff
@@ -140,12 +140,22 @@ func (p *protocol) Configure(_ transport.PeerID) error {
 func (p *protocol) Start() (err error) {
 	p.ctx, p.cancel = context.WithCancel(context.Background())
 
-	// load old payload query jobs
-	if err = p.payloadScheduler.Run(); err != nil {
-		return fmt.Errorf("failed to start retrying TransactionPayloadQuery: %w", err)
+	nodeDID, err := p.nodeDIDResolver.Resolve()
+	if err != nil {
+		log.Logger().WithError(err).Errorf("failed to resolve node DID")
```

Comment:
I think we (normally/should) always start log messages with a capital (errors start lowercase)